tests/test_states.py

```diff
@@ -220,6 +247,394 @@ def test_squeezed_coherent(self):
         self.assertAllAlmostEqual(res.flatten(), res_exact.flatten(), delta=self.tol)
 
 
+class PolyQuadExpectationSingleMode(BaseTest):
+    num_subsystems = 3
+    cutoff = 12
+    hbar = 2
+
+    # qudrature rotation
+    qphi = 0.78
+
+    # construct the expected vector of means and covariance matrix
+    mu = R(qphi).T @ np.array([a.real, a.imag]) * np.sqrt(2*hbar)
+    cov = R(qphi).T @ squeezed_cov(r, phi, hbar=hbar) @ R(qphi)
+
+    def sample_one_mode_normal_expectations(self, mu, cov, func):
+        """Returns the expectation value E(f) and the variance var(f)
+        for some normal distribution X~N(mu, cov).
+
+        Args:
+            mu (array): means vector
+            cov (array): covariance matrix
+            func (function): function acting on the random variables X, P, XP,
```

Comment:
only X, P, and XP?

Reply:
Because you can get X^2 from X, etc. :laughing: Maybe I was a bit lazy there